suite/tests/api/dis-a64.txt

```diff
@@ -1561,6 +1561,16 @@ fd3fffff : str    d31, [sp,#32760]        : str    %d31 -> +0x7ff8(%sp)[8byte]
 fd481041 : ldr    d1, [x2,#4128]          : ldr    +0x1020(%x2)[8byte] -> %d1
 fd7fffff : ldr    d31, [sp,#32760]        : ldr    +0x7ff8(%sp)[8byte] -> %d31
 
+
+# ADD (vector)
+4e2c856a : add v10.16b, v11.16b, v12.16b : add    %q11 %q12 $0x00 -> %q10
```

Comment:
The script `dis-a64.pl` contains a format specification that would attempt to make these colons line up. In fact, I think at some point `dis-a64.txt` could survive reformatting by `dis-a64.pl`. Perhaps worth thinking about getting that to work again, but not as part of this commit.

Reply:
Right, I've aligned the ADD lines. It should be easy to update the generator script to align on a per-opcode basis.